app/views/users/show.html.erb

```diff
@@ -1,15 +1,41 @@
-<h1>
-  <%= @user.name %>
-</h1>
+<table class="table">
+  <tbody>
+    <tr>
+      <td>
+        <%= image_tag @user.photo, alt: 'User Photo', class: 'user-photo' %>
+      </td>
+      <td>
+        <h1><%= @user.name %></h1>
+      </td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
- [ ] Kindly ensure that your design matches the given design in terms of elements and layouts to meet the requirements.

your design
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/0ddca974-b1d7-4c5e-9067-0db6eee13ab2)

given frame
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/8fc50fb6-171d-4680-bf2b-038e4a95f416)


- Kindly add the missing `Number of posts` in your design
- Use the `#recent_post` method in your user model to render the user's 3 most recent post.